Triangulation_3/doc/Triangulation_3/Doxyfile.in

```diff
@@ -1,3 +1,11 @@
 @INCLUDE = ${CGAL_DOC_PACKAGE_DEFAULTS}
 
 PROJECT_NAME = "CGAL ${CGAL_DOC_VERSION} - 3D Triangulations"
+
+
+INPUT                      =  ${CMAKE_SOURCE_DIR}/Triangulation_3/doc/Triangulation_3/
+
+
+EXTRACT_ALL                =  false
+HIDE_UNDOC_MEMBERS         =  true
+HIDE_UNDOC_CLASSES         =  true
```

Comment:
The issue is here: `${CMAKE_SOURCE_DIR}/Triangulation_3` is not a valid path in the tarballs layout.